apps/src/code-studio/pd/regional_partner_contact/RegionalPartnerContact.jsx

```diff
@@ -7,6 +7,31 @@ export default class RegionalPartnerContact extends FormController {
   /**
    * @override
    */
+  serializeFormData() {
+    const dataWithDistrict = {};
+    dataWithDistrict['form_data'] = Object.assign({}, super.serializeFormData()['form_data'], this.getDistrictData());
+
+    return dataWithDistrict;
+  }
+
+  getDistrictData() {
+    const schoolDistrictData = {};
+
+    schoolDistrictData['school-type'] = document.getElementById('school-type').value;
+    schoolDistrictData['school-state'] = document.getElementById('school-state').value;
+    schoolDistrictData['school-district'] = document.querySelector('#school-district input').value;
+    schoolDistrictData['school-district-other'] = document.getElementById('school-district-other').checked;
+    schoolDistrictData['school'] = document.querySelector('#school input').value;
+    schoolDistrictData['school-other'] = document.getElementById('school-other').checked;
+    schoolDistrictData['school-district-name'] = document.getElementById('school-district-name').value;
+    schoolDistrictData['school-name'] = document.getElementById('school-name').value;
+    schoolDistrictData['school-zipcode'] = document.getElementById('school-zipcode').value;
```

Comment:
how are these elements getting added to the page?

Reply:
https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/pd/regional_partner_contact/new.html.haml

These are elements in the school_info partial